bundles/core/src/main/java/com/adobe/cq/forms/core/components/models/form/Base.java

```diff
@@ -18,221 +18,23 @@
 import java.util.Collections;
 import java.util.Map;
 
-import javax.annotation.Nonnull;
-
-import org.apache.commons.lang3.StringUtils;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 import org.osgi.annotation.versioning.ConsumerType;
 
 import com.adobe.cq.export.json.ComponentExporter;
-import com.adobe.cq.forms.core.components.views.Views;
-import com.adobe.cq.wcm.core.components.models.Component;
-import com.day.cq.i18n.I18n;
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonInclude;
-import com.fasterxml.jackson.annotation.JsonInclude.Include;
-import com.fasterxml.jackson.annotation.JsonValue;
-import com.fasterxml.jackson.annotation.JsonView;
 
 /**
  * A base interface to be extended by all the different types of form elements.
  *
  * @since com.adobe.cq.forms.core.components.models.form 0.0.1
  */
 @ConsumerType
-public interface Base extends Component {
-
-    public final String CUSTOM_PROPERTY_WRAPPER = "af:layout";
+public interface Base extends FormComponent {
     public final String DATE_FORMATTER = "yyyy-MM-dd";
 
-    /**
-     * Defines the view type. Possible values: {@code text-input}, {@code multiline-input}, {@code number-input}, {@code date-input},
-     * {@code file-input}, {@code drop-down}, {@code radio-group}
-     * , {@code plain-text}, {@code checkbox}, {@code button}, {@code panel}
-     *
-     * @since com.adobe.cq.forms.core.components.models.form 0.0.1
-     */
-    public enum FieldType {
```

Comment:
Moving this to Base to FormComponent would require a major version bump (as it changes the bytecode in a backwards incompatible way). Hence we are moving it to a separate place only